pandas/core/generic.py

```diff
@@ -5664,7 +5664,7 @@ def _to_dict_of_blocks(self, copy=True):
             for k, v, in self._data.to_dict(copy=copy).items()
         }
 
-    def astype(self, dtype, copy=True, errors="raise"):
+    def astype(self, dtype, copy: bool_t = True, errors: str = "raise"):
```

Comment:
did you mean to use bool_t here?

Reply:
Yes.  We can't use `bool` here because NDFrame defines bool, see note on L142

Reply:
ok, interesting